var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -23,9 +23,13 @@ class Neuron(Package):
     url      = "http://www.neuron.yale.edu/ftp/neuron/versions/v7.5/nrn-7.5.tar.gz"
     git      = "https://github.com/nrnhines/nrn.git"
 
+    # Patch which reverts 81a7a39 for numerical compat
+    patch('revert_Import3d_numerical_format.patch', when='@7.8.0:')
+
     version('develop', branch='master')
+    version('7.8.1a',  commit='92a208b', preferred=True)
     version('7.8.0',   tag='7.8.0')
```

Comment:
As Omar pointed out, 7.8.0 is bit misnomer (and not officially released yet if iirc). We can also call this some 7.7.x if you guys prefer. (We haven’t deployed 7.8.0 I guess and the tag should be removed I think).

Reply:
7.7.3 might be a good option since the referred commit includes the commits from 7.7(.1) which is the latest release and 7.7.2 which is another tag already in neuron

Reply:
Here we are in muddy ground. It's gonna be confusing having a version 7.7.3 pointing to a commit after tag 7.8.0
I think we should keep the weird versioning scheme in mind, but in any case it shouldn't affect the matching here between versions and tags. Hopefully soon we start having proper tagging and branching in neuron
So I would suggest to remove the previous 7.8.0 version since it's not a 'real' version, and name this commit 7.8.0a. Does it sound reasonable?

Reply:
Ah the commit `92a208b` is after the "inital" commit of `7.8.0`. Okay then, `7.8.0a` makes sense to me :+1: